templates/zerver/api/changelog.md

```diff
@@ -20,8 +20,10 @@ format used by the Zulip server that they are interacting with.
 
 ## Changes in Zulip 6.0
 
-Feature levels 123-124 are reserved for future use in 5.x maintenance
-releases.
+**Feature level 125**
+
+* Replacing the previous `allow_edit_history` boolean field to
+  `message_edit_history_visibility` integer field.
```

Comment:
This should list the endpoints affected with links, like in other examples in this file. In general it's always worth looking for something similar to see the preferred style in a given part of the codebase,